File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for a in range(len(nums)):
            num = nums[a]
            search_int = target - num
            for b in range(len(nums)):
                if a != b and search_int == nums[b]:
                    return [a,b]

File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        std::unordered_map<int, bool> numMap;
        for (int num : nums) {
        // If the number is already in the map, we found a duplicate
            if (numMap.find(num) != numMap.end()) {
                return true;
            }
            // Otherwise, add the number to the map
            numMap[num] = true;
        }

    // If no duplicates were found, return false
    return false;
    }
};
